playwright-tests/utils/sovereignTransfer.ts

```diff
@@ -0,0 +1,18 @@
+import { Page } from '@playwright/test';
+import { DataTestIdsEnum } from '../../src/localConstants/dataTestIds.enum';
+import { AccountStatesEnum, GlobalDataEnum } from '../utils/enums.ts';
+
+interface SovereignTransfer {
```

Comment:
This should be called maybe `SovereignTransferType` or something indicating it's a type.